### DISCUSSION TITLE
[Editor] Image Rendering

### DISCUSSION THREAD

**Discussion by littpi** [livewire/flux#872] [Feature Requests]
I am using editor component to adopt to my old system and I found that the images of my articles cannot be rendered inside the editor. I have checked the Tiptap document and I think the `editor.js` does not import the image extension for rendering images. Will it be added in the upcoming version?

Reference: https://tiptap.dev/docs/editor/extensions/nodes/image

**Comment by NoisyFlake** [comment]
Yes, I'd also love a feature to allow adding images (either via button or drag-and-drop) right in the editor and then saving them as base64, so no server-interaction is required.

**Comment by mauritskorse** [comment]
Also see this discussion: https://github.com/livewire/flux/issues/746

**Comment by joshhanley** [comment]
Thanks for the request! As @mauritskorse noted, we have an open issue #746 about allowing people to add their own customisations to the editor component. So I'm going to close this feature request in favour of that issue. Please add your support to that issue. Thanks!
